XRTK-Core/Packages/com.xrtk.core/Inspectors/Utilities/MixedRealityInspectorUtility.cs

```diff
@@ -186,6 +188,27 @@ public static void CheckProfileLock(this BaseMixedRealityProfile target, bool di
             }
         }
 
+        /// <summary>
+        /// Checks if the profile can be inspected with the currently active build target and renders
+        /// a hint to switch build target if not.
+        /// </summary>
+        /// <param name="supportedPlatforms">The supported platforms by the profile.</param>
+        /// <param name="infoText">Optional info text to override info box text.</param>
+        /// <returns>True, if the profile and the active build target match.</returns>
+        public static bool CheckProfilePlatform(SupportedPlatforms supportedPlatforms, string infoText = null)
```

Comment:
I like this. Let's pull it out into it's own PR

Reply:
@provencher and I had a good discussion on this yesterday, should also be factored into the new Platform PR.  But using a simple bitwise enum